### PR TITLE
[wave2-700] initial fix for cloning with view’s ownership.

### PR DESCRIPTION
**Pull request by mhughes2012**

### CODE REVIEW

**Comment by balrajb** [comment]
@mhughes2012 as I refactored the content ownership module, can you please make move this code in "content_ownership_code" submodule.

File: sites/all/modules/ucalgary/content_ownership/content_ownership_core/views_ownership.inc [hunk 1]
```diff
@@ -409,6 +410,29 @@ function _views_ownership_build_ownership_fieldset($form, $form_state, $user_gro
...
+  if ($clonedpage ) {
+
+    // clear the owners array
+    $existing_owners = array();
+
+    global $user;
+    $user = user_load($user->uid);
+
+      // get this user's group ownership
+      $ug = $user->field_user_ownership_group[und];
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/content_ownership/content_ownership_core/views_ownership.inc]: und should be defined as 'und'

File: sites/all/modules/ucalgary/content_ownership/content_ownership_core/views_ownership.inc [hunk 2]
```diff
@@ -409,6 +410,29 @@ function _views_ownership_build_ownership_fieldset($form, $form_state, $user_gro
         }
     }
 
+  $clonedpage =  basename($_SERVER['HTTP_REFERER']) == 'clone';
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/content_ownership/content_ownership_core/views_ownership.inc]: Remove extra spaces between equals and basename

File: sites/all/modules/ucalgary/content_ownership/content_ownership_core/views_ownership.inc [hunk 3]
```diff
@@ -409,6 +410,29 @@ function _views_ownership_build_ownership_fieldset($form, $form_state, $user_gro
...
+
+
+  if ($clonedpage ) {
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/content_ownership/content_ownership_core/views_ownership.inc]: Remove extra spacing around close between variable and close bracket

File: sites/all/modules/ucalgary/content_ownership/content_ownership_core/views_ownership.inc [hunk 4]
```diff
@@ -409,6 +410,29 @@ function _views_ownership_build_ownership_fieldset($form, $form_state, $user_gro
...
+
+    global $user;
+    $user = user_load($user->uid);
+
+      // get this user's group ownership
+      $ug = $user->field_user_ownership_group[und];
+
+      // the key is 'target_id', so using a simple counter for index of new array
+      $count = 0;
+      foreach ($ug as $term ) {
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/content_ownership/content_ownership_core/views_ownership.inc]: Remove extra spacing around close between variable and close bracket

File: sites/all/modules/ucalgary/content_ownership/content_ownership_core/views_ownership.inc (hunk 3 above)
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/content_ownership/content_ownership_core/views_ownership.inc]: Remove extra line space on 415

File: sites/all/modules/ucalgary/content_ownership/content_ownership_core/views_ownership.inc (hunk 2 above)
Comment (mhughes2012) [inline sites/all/modules/ucalgary/content_ownership/content_ownership_core/views_ownership.inc]: done.

File: sites/all/modules/ucalgary/content_ownership/content_ownership_core/views_ownership.inc (hunk 3 above)
Comment (mhughes2012) [inline sites/all/modules/ucalgary/content_ownership/content_ownership_core/views_ownership.inc]: done.

File: sites/all/modules/ucalgary/content_ownership/content_ownership_core/views_ownership.inc (hunk 1 above)
Comment (mhughes2012) [inline sites/all/modules/ucalgary/content_ownership/content_ownership_core/views_ownership.inc]: done.

File: sites/all/modules/ucalgary/content_ownership/content_ownership_core/views_ownership.inc (hunk 4 above)
Comment (mhughes2012) [inline sites/all/modules/ucalgary/content_ownership/content_ownership_core/views_ownership.inc]: done.

**Pull request merged**